app/views/admin/infobase/_order_form.html.haml

```diff
@@ -0,0 +1,24 @@
+%table.ui.very.basic.table{id: 'order-table'}
+  %thead
+    %tr
+      %th User Name
+      %th User Email
+      %th Delivery Address
+      %th.center.aligned Order Items
+      %th Date of Order
+      %th.right.aligned Price of Order
+      %th
+  %tbody
+  - orders.each do |order|
+    %tr
+      %td #{order.user_name}
```

Comment:
wtf :)
`%td= order.user_name`

also you can play around with `each` and `send` here, it's always preferable if columns don't have unique properties